### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by St4rz**
Correct instances of "retriver" to "retriever"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -460,12 +460,12 @@ libraryJob.run()
         libraryJob.libraryResource(net/courtanet/jenkins/request.json)
         libraryJob.sh(curl -H 'Content-Type: application/json' -X POST -d '{"name" : "Ben"}' http://acme.com)
 ```
-### Library Source Retrivers
+### Library Source Retriever
```
Comment (stchar) [inline README.md]: Library Source Retriever**s**

**Comment by stchar** [comment]
Hi, thanks a lot, just a minor correction is needed. Would you be so kind ;)

File: README.md (hunk 1 above)
Comment (St4rz) [reply README.md]: Oops, thanks for pointing that out!

**Comment by stchar** [comment]
thank you, merged.

**Pull request merged**